code/modules/assembly/doorcontrol.dm

```diff
@@ -13,6 +13,12 @@
 	if(id)
 		. += "<span class='notice'>Its channel ID is '[id]'.</span>"
 
+/obj/item/assembly/control/multitool_act(mob/living/user)
```

Comment:
Here too, does this let me just screw with the shutter doors in security just by changing the ID of a completely separated button?

Also, if you're doing this for construction purposes, doesn't being unable to create these defeat the point? Or can you create these already?

Reply:
Oh yeah you can create them in an autolathe

Reply:
And yeah you could screw with them, but you can already do that without my pr, just screwdrive the button and take them out